src/low_points.ts

```diff
@@ -0,0 +1,44 @@
+export function isLowPoint(prev: number, cur: number, next: number): boolean {
+  return (prev > cur) && (next > cur);
+}
+
+// Assumes that the grid has a consistent width and height.
+export function locateLowPointsInGrid(grid: number[][]): [number, number][] {
+  let maximum = 9; // TODO: autogenerate this
+  let paddingHorizontal: number[] = Array(grid[0].length).fill(maximum);
+
+  // Builds a grid with padding around it
+  grid = [paddingHorizontal].concat(grid).concat([paddingHorizontal]);
+  grid = grid.map(line => [maximum].concat(line).concat([maximum]));
+
+  let locationsInPaddedGrid = [];
+  // Start at 1 to not check padding
+  for (let x = 1; x < grid[0].length - 1; x++) {
+    for (let y = 1; y < grid.length - 1; y++) {
+      type WindowOfPoints = [number, number, number]
```

Comment:
Can you extract this into an interface?  It would help to improve readability for the `isLowPoint` function.